extensions/amp-pan-zoom/amp-pan-zoom.md

```diff
@@ -51,5 +51,17 @@ limitations under the License.
   </amp-layout>
 ```
 
-## Events
-It exposes the `zoomEnd` action, which contains the single value, `scale`. This value contains the current scale of the child content being zoomed.
\ No newline at end of file
+## Attributes
+##### max-scale (optional)
+Specifies a max zoom scale, otherwise set to 3.
```

Comment:
nit, up to you: specify it should be a whole number (same below).

Reply:
Excellent catch actually, `max-scale` and `initial-scale` should allow decimals. Changed regex to `"[0-9]+\.[0-9]+"`. Does that look right?

Reply:
This will require the `.` but maybe that's okay `0.0` or `1.0`. Also note that escapes need to be double escaped, so make `\.` into `\\.`

Reply:
Okay, this should be correct now. `[0-9]+(\.[0-9]+)?`